src/App.svelte

```diff
@@ -1,372 +1,26 @@
-					  "Liste de Choses à Faire",
-					  "Lista di Cose da Fare",
-					  "Lista de Afazeres",
-					  "Список Дел",
-					  "待办事项",
-					  "タスク一覧",
-					  "רשימת המטלות",
-					  "🍈🍈🍈" //get it, honeydew? honey do? i'm hilarious...
-					  ];
-		let index = Math.floor(Math.random() * titles.length);
-		return titles[index];
-	}
-
-	$: title = selectRandomTitle();
-
-	// function to toggle alerts
-	function toggleAlerts() {
-		alertsOn = !alertsOn;
-	}
+  // flag to handle alert status
+  let alertsOn = true;
 </script>
 
-<body>
-	<main>
-		<h1>{title}</h1>
-		<h2>
-			<table class="centered">
-				<td>
-					<ToolTip title="Type something that needs to get done, then press Enter or click ✔️ below to add it to the list.">
-						<input name="inputField1" class="italic" type="text" bind:this={inputField} bind:value={input} placeholder="What needs to be done?" on:keypress={onKeyPress} size="25">
-					</ToolTip>
-				</td>
-				<td>
-					<ToolTip title="Click to add this new task to the list.">
-						<button on:click={addTask(inputField.value)}><Emoji symbol="✔️" label="Add Task"></Emoji></button>
-					</ToolTip>
-				</td>
-			</table>
-		</h2>
-		
-		<div class="pane">
-			<table class="main">
-				{#if mode === "all"}
-					{#each tasks as task}
-						<tr class:done={task.done}>
-							<td>
-								<ToolTip title="Click to toggle whether this task has been completed.">
-									<button on:click={completeTask(task)}><Emoji symbol="🏆" label="Completed"></Emoji></button>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to edit this task.">
-									<td contenteditable="true" bind:innerHTML={task.text}/>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to delete this task from the list.">
-									<button on:click={() => deleteTask(task)}><Emoji symbol="🗑️" label="Delete"></Emoji></button>
-								</ToolTip>
-							</td>
-						</tr>
-					{/each}
-				{:else if mode === "active"}
-					{#each activeTasks as task}
-						<tr class:done={task.done}>
-							<td>
-								<ToolTip title="Click to toggle whether this task has been completed.">
-									<button on:click={completeTask(task)}><Emoji symbol="🏆" label="Completed"></Emoji></button>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to edit this task.">
-									<td contenteditable="true" bind:innerHTML={task.text}/>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to delete this task from the list.">
-									<button on:click={() => deleteTask(task)}><Emoji symbol="🗑️" label="Delete"></Emoji></button>
-								</ToolTip>
-							</td>
-						</tr>
-					{/each}
-				{:else}
-					{#each completedTasks as task}
-						<tr class:done={task.done}>
-							<td>
-								<ToolTip title="Click to toggle whether this task has been completed.">
-									<button on:click={completeTask(task)}><Emoji symbol="🏆" label="Completed"></Emoji></button>
```

Comment:
I'm actually surprised the `on:click` worked here. Normally you'd have to do `on:click={() => completeTask(task)}`. Unless Svelte is doing some sorcery 🤷 .